tests/phpunit/CRM/Contribute/BAO/ContributionRecurTest.php

```diff
@@ -99,15 +99,15 @@ public function testCancelRecur() {
    */
   public function testSupportFinancialTypeChange() {
     $contributionRecur = $this->callAPISuccess('contribution_recur', 'create', $this->_params);
-    $contribution = $this->callAPISuccess('contribution', 'create', array(
+    $this->callAPISuccess('contribution', 'create', array(
```

Comment:
these changes are unrelated - I noticed these minor errors while looking in the wrong file